sdk/storage/azure-storage-files-shares/inc/azure/storage/files/shares/rest_client.hpp

```diff
@@ -2210,7 +2210,7 @@ namespace Azure { namespace Storage { namespace Files { namespace Shares {
         std::string FileAttributes;
         Nullable<std::string> FileCreationTime;
         Nullable<std::string> FileLastWriteTime;
-        Nullable<DateTime> FileChangeTime;
+        Nullable<std::string> FileChangeTime;
```

Comment:
Would this change be a breaking in interface?

Reply:
No, it's all in detail namespace, and we're still working on this feature.